dist.ini

```diff
@@ -95,6 +95,8 @@ skip = ^MIME::Lite$
 perl                   = 5.008
 Regexp::Common         = 2013031301
 Config::Tiny           = 0
+Date::Format           = 0
+Date::Parse            = 0
 File::ShareDir         = 0
```

Comment:
these should move down to the [Prereqs / RuntimeRecommends] as they are are only needed for DMARC reporting (not validation).